src/rules.jl

```diff
@@ -5,46 +5,46 @@
 @define_diffrule Base.cbrt(x)                 = :(  inv(3 * cbrt($x)^2)                )
 @define_diffrule Base.abs2(x)                 = :(  $x + $x                            )
 @define_diffrule Base.inv(x)                  = :( -abs2(inv($x))                      )
 @define_diffrule Base.log(x)                  = :(  inv($x)                            )
-@define_diffrule Base.log10(x)                = :(  inv($x) / log(10)                  )
-@define_diffrule Base.log2(x)                 = :(  inv($x) / log(2)                   )
+@define_diffrule Base.log10(x)                = :(  inv($x) / $logten      )
+@define_diffrule Base.log2(x)                 = :(  inv($x) / $logtwo      )
 @define_diffrule Base.log1p(x)                = :(  inv($x + 1)                        )
 @define_diffrule Base.exp(x)                  = :(  exp($x)                            )
-@define_diffrule Base.exp2(x)                 = :(  exp2($x) * log(2)                  )
-@define_diffrule Base.exp10(x)                = :(  exp10($x) * log(10)                )
+@define_diffrule Base.exp2(x)                 = :(  exp2($x) * $logtwo     )
+@define_diffrule Base.exp10(x)                = :(  exp10($x) * $logten    )
 @define_diffrule Base.expm1(x)                = :(  exp($x)                            )
 @define_diffrule Base.sin(x)                  = :(  cos($x)                            )
 @define_diffrule Base.cos(x)                  = :( -sin($x)                            )
 @define_diffrule Base.tan(x)                  = :(  1 + tan($x)^2                      )
 @define_diffrule Base.sec(x)                  = :(  sec($x) * tan($x)                  )
 @define_diffrule Base.csc(x)                  = :( -csc($x) * cot($x)                  )
 @define_diffrule Base.cot(x)                  = :( -(1 + cot($x)^2)                    )
-@define_diffrule Base.sind(x)                 = :(  (π / 180) * cosd($x)               )
-@define_diffrule Base.cosd(x)                 = :( -(π / 180) * sind($x)               )
-@define_diffrule Base.tand(x)                 = :(  (π / 180) * (1 + tand($x)^2)       )
-@define_diffrule Base.secd(x)                 = :(  (π / 180) * secd($x) * tand($x)    )
-@define_diffrule Base.cscd(x)                 = :( -(π / 180) * cscd($x) * cotd($x)    )
-@define_diffrule Base.cotd(x)                 = :( -(π / 180) * (1 + cotd($x)^2)       )
+@define_diffrule Base.sind(x)                 = :(  π * cosd($x) / 180                 )
+@define_diffrule Base.cosd(x)                 = :( -(π * sind($x) / 180)               )
+@define_diffrule Base.tand(x)                 = :(  π * (1 + tand($x)^2) / 180         )
+@define_diffrule Base.secd(x)                 = :(  π * secd($x) * tand($x) / 180      )
+@define_diffrule Base.cscd(x)                 = :( -(π * cscd($x) * cotd($x) / 180)    )
+@define_diffrule Base.cotd(x)                 = :( -(π * (1 + cotd($x)^2) / 180)       )
```

Comment:
Can we use `deg2rad` as in https://github.com/JuliaDiff/ChainRules.jl/pull/554?

```suggestion
@define_diffrule Base.sind(x)                 = :(  deg2rad(cosd($x))  )
@define_diffrule Base.cosd(x)                 = :( - deg2rad(sind($x))   )
@define_diffrule Base.tand(x)                 = :(  deg2rad(1 + tand($x)^2)  )
@define_diffrule Base.secd(x)                 = :(  deg2rad(secd($x) * tand($x))  )
@define_diffrule Base.cscd(x)                 = :( - deg2rad(cscd($x) * cotd($x))  )
@define_diffrule Base.cotd(x)                 = :( - deg2rad(1 + cotd($x)^2)  )
```